src/qibolab/instruments/qblox/port.py

```diff
@@ -273,15 +259,11 @@ def lo_frequency(self, value):
         self._settings.lo_frequency = value
         if self.module.device:
             if self.module.device.is_qrm_type:
-                self.module._set_device_parameter(
-                    self.module.device,
-                    f"out{self.port_number}_in{self.port_number}_lo_freq",
-                    value=value,
+                self.module.device.set(
+                    f"out{self.port_number}_in{self.port_number}_lo_freq", value=value
                 )
             elif self.module.device.is_qcm_type:
-                self.module._set_device_parameter(
-                    self.module.device, f"out{self.port_number}_lo_freq", value=value
-                )
+                self.module.device.set(f"out{self.port_number}_lo_freq", value=value)
         else:
             pass
             # TODO: This case regards a connection error of the module
```

Comment:
We could probably drop this completely (`if` does not always need to have `else`)
```suggestion
```